boards/esp32s3-box/Makefile.dep

```diff
@@ -1,15 +1,3 @@
-ifeq (,$(filter stdio_% slipdev_stdio usbus usbus% tinyusb_device,$(USEMODULE)))
-  # Use stdio_usb_serial_jtag if no other stdio is requested explicitly
-  # and neither USBUS nor tinyusb_device are used
-  USEMODULE += stdio_usb_serial_jtag
-  # Even if only stdio_usb_serial_jtag is enabled, usb_board_reset is enabled
-  # since there should be a CDC ACM interface in any case. This is necessary,
-  # for example, to reset the board if stdio_cdc_acm or stdio_tinyusb_cdc_acm
-  # was previously used.
-  USEMODULE += usb_board_reset
-  # include $(RIOTMAKE)/tools/usb_board_reset.mk
```

Comment:
This was enabled on some boards and disabled on some others - I moved it to a common Makefile and enabled it for all.